modules/api_gateway/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "tags" {
+  default     = {}
```

Comment:
Let's keep the way we define variables consistent... i.e., if the type is defined first for one then define it first for all.

Reply:
fixed